home/whatsapp.py

```diff
@@ -61,7 +61,7 @@ def get_upload_session_id(image_id):
     img_obj = get_image_model().objects.get(id=image_id)
     mime_type = mimetypes.guess_type(img_obj.file.name)[0]
     file_size = img_obj.file.size
-    file_path = img_obj.file.path
+    file_path = Path(settings.MEDIA_ROOT) / img_obj.file.name
```

Comment:
I'm not sure this will work either. This is still looking on a local disk for the file, not downloading it from S3. We probably want to open it directly, instead of opening a path to the file: https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#storage